cloudbuild-pr.yaml

```diff
@@ -81,6 +81,17 @@ steps:
       ../gcs-service-account.json
       $PROJECT_ID
 
+- name: 'gcr.io/cloud-builders/kubectl'
+  id: run-migrations
+  waitFor: ['helm-deploy-studio-instance']
+  env:
+  - 'CLOUDSDK_COMPUTE_ZONE=us-central1-f'
+  - 'CLOUDSDK_CONTAINER_CLUSTER=dev-qa-cluster'
+  args:
+  - create
+  - job
+  - --from=cronjob/$_RELEASE_NAME-studio-job-template
```

Comment:
just wondering, why do we use cron job instead of job here?

Reply:
Moved to a Job. Looks like it's better overall.